src/components/Discover.jsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import DestinationCard from './DestinationCard';
+import './Discover.css';
+import './DestinationCard.css';
+import Wellness from './Wellness';
+import WellnessImage from '../../images/WellnessImage.jpg';
+import Romantic from './Romantic'
+import RomanticImg from '../../images/RomanticImg.jpg'
+import FamilyFriendly from './FamilyFriendly';
+import FamilyFriendlyImage from '../../images/FamilyFriendlyImage.jpg'
+import BudgetTravel from './BudgetTravel';
+import BudgetTravelImage from '../../images/BudgetTravel.jpg'; 
```

Comment:
Import is pathing to BudgetTravel.jpg, but in /images the image is named Budgettravel.jpg